src/Card/CardSection/index.js

```diff
@@ -90,12 +91,34 @@ class CardSection extends React.Component<any, Props> {
     }
   };
 
+  handleKeyPress = (ev: SyntheticKeyboardEvent<HTMLInputElement>) => {
+    if (ev.key === "Enter" || ev.key === " ") {
+      ev.preventDefault();
+      this.injectCallbackAndToggleSection();
+
+      const { onClose, onExpand, expanded } = this.props;
```

Comment:
This code below does do the same as `injectCallbackAndToggleSection` please remove 🙏